src/supertux/main.cpp

```diff
@@ -265,20 +264,20 @@ std::string olduserdir = FileSystem::join(physfs_userdir, PACKAGE_NAME);
 std::string olduserdir = FileSystem::join(physfs_userdir, "." PACKAGE_NAME);
 #endif
 if (FileSystem::is_directory(olduserdir)) {
-  boost::filesystem::path olduserpath(olduserdir);
-  boost::filesystem::path userpath(userdir);
+  std::filesystem::path olduserpath(olduserdir);
+  std::filesystem::path userpath(userdir);
 
-  boost::filesystem::directory_iterator end_itr;
+  std::filesystem::directory_iterator end_itr;
 
   bool success = true;
 
   // cycle through the directory
-  for (boost::filesystem::directory_iterator itr(olduserpath); itr != end_itr; ++itr) {
+  for (std::filesystem::directory_iterator itr(olduserpath); itr != end_itr; ++itr) {
```

Comment:
`filesystem::directory_iterator` can be used directly with range-based `for`:

```suggestion
  for (auto file : std::filesystem::directory_iterator(olduserpath)) {
```